test/shopify/scopes_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'omniauth-shopify-oauth2'
+
+class ScopesTest < Minitest::Test
```

Comment:
If folks are going to use this outside of the gem then I would fill out the unit tests here.